src/components/Menu.js

```diff
@@ -10,17 +10,23 @@ const refresh = new URL('../assets/icons/refresh.svg', import.meta.url).href;
 const add = new URL('../assets/icons/circle-plus.svg', import.meta.url).href;
 
 const renderMenu = menuItem => {
+  const output = {};
   switch (menuItem) {
     case 'info':
-      return html`
+      output.title = 'info';
+      output.template = html`
         info
       `;
+      break;
```

Comment:
Why not `return { title: 'info', template: html`...`  }` For all of these?

Reply:
much better 👍